projects/1.16/assets/mystical-world/mysticalworld/lang/zh_cn.json

```
@@ -1 +1,434 @@
+    "item.mysticalworld.nautilus_horn": "鹦鹉螺角",
+    "item.mysticalworld.owl_spawn_egg": "猫头鹰刷怪蛋",
+    "item.mysticalworld.pearleporter": "珍珠提取器",
+    "item.mysticalworld.pelt": "毛皮",
+    "item.mysticalworld.peony_cordial": "牡丹果汁",
+    "item.mysticalworld.quicksilver_axe": "水银斧",
+    "item.mysticalworld.quicksilver_boots": "水银靴子",
+    "item.mysticalworld.quicksilver_chestplate": "水银胸甲",
+    "item.mysticalworld.quicksilver_dust": "水银粉",
+    "item.mysticalworld.quicksilver_helmet": "水银头盔",
+    "item.mysticalworld.quicksilver_hoe": "水银锄",
+    "item.mysticalworld.quicksilver_ingot": "水银锭",
+    "item.mysticalworld.quicksilver_knife": "水银刀",
+    "item.mysticalworld.quicksilver_leggings": "水银护腿",
+    "item.mysticalworld.quicksilver_nugget": "水银粒",
+    "item.mysticalworld.quicksilver_pickaxe": "水银镐",
+    "item.mysticalworld.quicksilver_shovel": "水银锹",
+    "item.mysticalworld.quicksilver_spear": "水银矛",
+    "item.mysticalworld.quicksilver_sword": "水银剑",
+    "item.mysticalworld.raw_squid": "生鱿鱼",
+    "item.mysticalworld.rose_cordial": "玫瑰果汁",
+    "item.mysticalworld.sapphire": "蓝宝石",
+    "item.mysticalworld.sapphire_axe": "蓝宝石斧",
+    "item.mysticalworld.sapphire_boots": "蓝宝石靴子",
+    "item.mysticalworld.sapphire_chestplate": "蓝宝石胸甲",
+    "item.mysticalworld.sapphire_helmet": "蓝宝石头盔",
+    "item.mysticalworld.sapphire_hoe": "蓝宝石锄",
+    "item.mysticalworld.sapphire_knife": "蓝宝石刀",
+    "item.mysticalworld.sapphire_leggings": "蓝宝石护腿",
+    "item.mysticalworld.sapphire_pickaxe": "蓝宝石镐",
+    "item.mysticalworld.sapphire_shovel": "蓝宝石锹",
+    "item.mysticalworld.sapphire_spear": "蓝宝石矛",
+    "item.mysticalworld.sapphire_sword": "蓝宝石剑",
+    "item.mysticalworld.silk_cocoon": "蚕茧",
+    "item.mysticalworld.silk_thread": "丝线",
+    "item.mysticalworld.silkworm_egg": "蚕卵",
+    "item.mysticalworld.silkworm_spawn_egg": "蚕刷怪蛋",
+    "item.mysticalworld.silver_axe": "银斧",
+    "item.mysticalworld.silver_boots": "银靴子",
+    "item.mysticalworld.silver_chestplate": "银胸甲",
+    "item.mysticalworld.silver_dust": "银粉",
+    "item.mysticalworld.silver_fox_spawn_egg": "银狐刷怪蛋",
+    "item.mysticalworld.silver_helmet": "银头盔",
+    "item.mysticalworld.silver_hoe": "银锄",
+    "item.mysticalworld.silver_ingot": "银锭",
+    "item.mysticalworld.silver_knife": "银刀",
+    "item.mysticalworld.silver_leggings": "银护腿",
+    "item.mysticalworld.silver_nugget": "银粒",
+    "item.mysticalworld.silver_pickaxe": "银镐",
+    "item.mysticalworld.silver_shovel": "银锹",
+    "item.mysticalworld.silver_spear": "银矛",
+    "item.mysticalworld.silver_sword": "银剑",
+    "item.mysticalworld.sliced_carrot": "胡萝卜条",
+    "item.mysticalworld.spindle": "纺锤",
+    "item.mysticalworld.sprout_spawn_egg": "芽精刷怪蛋",
+    "item.mysticalworld.stewed_eggplant": "炖茄子",
+    "item.mysticalworld.stone_knife": "石刀",
+    "item.mysticalworld.stone_spear": "石矛",
+    "item.mysticalworld.stuffed_aubergine": "填馅茄子",
+    "item.mysticalworld.tannin_vial": "丹宁酸瓶",

```

```suggestion
    "item.mysticalworld.tannin_vial": "单宁酸瓶",
```
```suggestion
    "item.mysticalworld.tannin_vial": "鞣酸瓶",
```
不过这几种貌似都有用到，个人身边统计学的话这两种貌似用得多些